### ISSUE TITLE
Add Scylla to benchmark

### ISSUE THREAD

**Issue by arothuis** [timescale/tsbs#5]
[Scylla][https://www.scylladb.com/] is intended to be a more performant, drop-in replacement for Cassandra.

It has been [benchmarked a bunch][https://www.scylladb.com/product/benchmarks/], but it would be interesting to see in this benchmark as well. The scripts for Cassandra code can be copied and modified slightly to fit Scylla.

**Comment by LeeHampton** [comment]
Agreed @arothuis -- I wanted to add Scylla (maybe KairosDB as well) when I was doing Cassandra benchmarks but didn't have time. Depending on whether the Scylla wire protocol and/or data model has significant differences from Cassandra, we may actually be able to just extend the existing Cassandra packages with a Scylla option (e.g., "engine=scylla" or something). 

In theory the data/query generation packages shouldn't need to be touched, but I'm not 100 % sure there are no differences in the way data is loaded/queried.
